hubgrep/models/repository.py

```diff
@@ -11,11 +12,16 @@
 
 class Repository(db.Model):
     __tablename__ = "repositories"
-    __table_args__ = (Index("repo_index", "foreign_id", "hosting_service_id"),)
 
-    id = db.Column(db.Integer, primary_key=True)
-    # set to true when import is finished
-    imported = db.Column(db.Boolean, nullable=True, default=False)
+    __table_args__ = (
+        #Index("hosting_service_id_index", "hosting_service_id"),
+        # UNLOGGED prevents wal write (as we will never edit the table, it should be fine?)
+        {"prefixes": ["UNLOGGED"]},
+    )
+
+    # use identity to avoid reusing same sequence when we copy this table:
+    # https://stackoverflow.com/a/12265248
+    id = db.Column(db.Integer, Identity(), primary_key=True)
```

Comment:
that was a tricky one. usually, a primary key creates a sequence, which has to be globally unique. so, when we copied this table, we also got the reference to the sequence copied, but we didnt create a new one.
since pg11 (..or so?) there is "identity", doing the same, but its tied to the table column.